### DISCUSSION TITLE
GitHub discussions or mailing lists for Uyuni for async discussions?

### DISCUSSION THREAD

**Discussion by juliogonzalez** [uyuni-project/uyuni#6821] [Polls]
Since @renner enabled GitHub discussions in 2020, we have both systems.

But that means everyone is duplicating the effort, as both places need to be checked to see if the question you want to ask is there, or if there was a reply.

Therefore, we need to pick one system and close the other one.

- Current list of mailing lists is at https://www.uyuni-project.org/pages/contact.html#ml
- GitHub discussions are available at https://github.com/uyuni-project/uyuni/discussions

The poll will be open for 2 weeks, until April 19th 17.00 CEST

**Comment by digdilem** [comment]
tbh, I didn't even know Discussions existed. But it looks good and clear people have been using it, and it gives a more searchable reference than a mailing list does, so voting for change.

**Comment by ppanon2022** [comment]
Both have their respective advantages. I find the mailing list useful with an autofilter/filer rule for my mailbox in that I can easily see what new subjects/emails have been added since I last checked the list. So that's more convenient for me. But for someone who wants to search for information on a subject, I can see the Github discussions being more useful and easy to search. On the other hand, switching to solely using GitHub may make it harder to find the right discussion because there's a lot of exchanges on the users mailing list for fleeting version-specific errors which would pollute the search space in Github Discussions if that was solely used.

**Reply by SchoolGuy** [reply]
For that purpose, you can label GitHub Discussion. That cannot be done by yourself but with the right search query, you should be able to find the discussions you want.

**Comment by bittin** [comment]
*voted*

**Comment by jcayouette** [comment]
Voted!

**Comment by wombelix** [comment]
Voted, but it's a difficult topic.

I feel that mails and an archive of them hosted directly by the project is more future proof. But it's also not much used anymore and lot of conversations get lost in chats these days, I guess that will not change when moving to GH discussions.

Also, right now GitHub is incredible successful and the largest platform, but long time ago, it was sourceforge.
What will happen with the content in GH discussions if another platform becomes popular one day? Risk is high that it just get lost at some point.

Mails and a separated mail archive will most likely last forever, see the kernel mailing list as example.

**Comment by BerndHelber** [comment]
With all honesty.
From my personal point of view. Only Relying on Github  might be a mistake, cause you have to trust Github, Platforms come and go. I know the days wenn RCS was suceeded by cvs, few years later by subversion, then came git. Those tools came and went donw the bin. Maybe we have different platforms in a few years, and another migration.  If you make use of mailing lists, you're more independent. This is a question of risk management. Mailing lists may not be the hottest thing in town nowadays. But they work and are  really simple to use.  The additional benefit that mails are searchable in the local mail client i really would miss.
Cheers

**Comment by juliogonzalez** [comment]
Deadline reached. Closing the poll.

Majority of votes go to moving to discussions, so I will act according to the community decision.

Plan is:
- Announce the decision at the announce mailing list, including the date when we'll archive lists
- Prepare a PR for the site
- Archive the lists
- Announce the archival is effected at the announce mailing list.
